opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -234,23 +254,34 @@ namespace Opm {
 
         static bool hasEqualDVDEPTHZ(const Deck&);
         static bool allEqual(const std::vector<double> &v);
+        std::vector<EclipseGridLGR> lgr_children_cells;
+
+    protected:
+        std::size_t lgr_global_counter = 0;
+        std::string lgr_label = "GLOBAL";
+        int lgr_level = 0;
+        std::vector<std::string> lgr_children_labels;
+        std::vector<std::size_t> lgr_active_index;
+        std::vector<std::size_t> lgr_level_active_map;
```

Comment:
Could you add some documentation about lgr_level_active_map? From line 1995 in EclipseGrid.cpp, it seems like lgr_level_active_map[active cell index] stores the level of the cell—is that correct? Is it then related to lgr_level? 

Also, could you provide some clarification for num_lgr_children_cells as well?